### ISSUE TITLE
Altme seems to mandate optional filter property in Presentation Exchange Input Descriptor

### ISSUE THREAD

**Issue by jfelixh** [TalaoDAO/AltMe#2769]
The DIF Specification (https://identity.foundation/presentation-exchange/#input-descriptor-object) states that the filter property is optional. Using a constraint without filter would be beneficial to just require a claim to be present without specific constraints on its value.

We would like to rely on this behavior to help users to see pre-selected credentials in Altme for our sign-in solution, the [SSI-to-OIDC-Bridge](https://github.com/GAIA-X4PLC-AAD/ssi-to-oidc-bridge). Our bridge configuration (at minimum) requires specified claims to be present, and that is what we would like to enforce via input descriptor.

## Expected Behavior

```json
[
  {
    "id": "emailpass_1",
    "name": "Input descriptor for credential 1",
    "purpose": "Sign-in",
    "constraints": {
      "fields": [
        {
          "path": ["$.credentialSubject.email"]
        }
      ]
    }
  }
]
```

Given this standard-compliant descriptor, Altme should present the user with all their email VCs. The user then selects one and presents it.

The descriptors we use were tested and verified with this library: https://github.com/Sphereon-Opensource/PEX

## Actual Behavior

Altme aborts the OID4VP presentation process, saying an error has occurred.

## Current Situation

We sidestep the issue with creative regex use. The descriptors we generate look like this and are accepted by Altme:

```json
[
  {
    "id": "emailpass_1",
    "name": "Input descriptor for credential 1",
    "purpose": "Sign-in",
    "constraints": {
      "fields": [
        {
          "path": ["$.credentialSubject.email"],
           "filter": {
              "type": "string",
              "pattern": "^.*$"
           }
        }
      ]
    }
  }
]
```

**Comment by jfelixh** [comment]
@jdsika fyi

**Comment by jdsika** [comment]
thx for the ping. Can you add the "ascs" label yourself?

**Comment by jtdemer** [comment]
Hi @ThierryThevenet the problem of @jfelixh is also very relevant for our customized wallet for the ascs. Can you please priotize this issue?

**Comment by ThierryThevenet** [comment]
@hawkbee1

**Comment by ThierryThevenet** [comment]
OIDC4VCI test 4

**Comment by ThierryThevenet** [comment]
fixed in 2.9.0
